bip-0039/german.txt

```diff
@@ -0,0 +1,2048 @@
+ABDRUCK
+ABEND
+ABFALL
+ABFLUG
+ABFOLGE
+ABGAS
+ABGLEICH
+ABHANG
+ABITUR
+ABKOMMEN
+ABMARSCH
+ABONNENT
+ABREISE
+ABSCHIED
+ABSEITS
+ABSICHT
+ABSPANN
+ABSTRICH
+ABTEIL
+ABWASCH
+ABZWEIG
+ACHSEL
+ACHTUNG
+ACKERBAU
+ADAPTER
+ADDITION
+ADJEKTIV
+ADJUDANT
+ADLER
+ADOPTION
+AFFE
+AFRIKA
+AGENDA
+AHNUNG
+AHORN
+AKADEMIE
+AKAZIE
+AKKORD
+AKTE
+AKTION
+AKUSTIK
+AKZENT
+ALCHEMIE
+ALIMENTE
+ALLEE
+ALLIANZ
+ALLMACHT
+ALLRAD
+ALLTAG
+ALPAKA
+ALPEN
+ALPHABET
+ALTERTUM
+ALTSTADT
+ALTWAGEN
```

Comment:
Not too commonly used, 300k Google results vs 10m+ for "Neuwagen" (which Altwagen would be the opposite of)

Reply:
Word replaced